coordinator/src/db/positions.rs

```diff
@@ -89,6 +89,32 @@ impl Position {
         Ok(positions)
     }
 
+    /// Get all active positions that were open before `open_before_timestamp`.
+    ///
+    /// Active positions are either [`PositionState::Open`], [`PositionState::Rollover`] or
+    /// [`PositionState::Resizing`].
+    pub fn get_all_active_positions_open_before(
+        conn: &mut PgConnection,
+        open_before_timestamp: OffsetDateTime,
+    ) -> QueryResult<Vec<crate::position::models::Position>> {
+        let positions = positions::table
+            .filter(
+                positions::position_state
+                    .eq(PositionState::Open)
+                    .or(positions::position_state.eq(PositionState::Rollover))
```

Comment:
Because positions that are being rolled over or resized are still active for the purposes of charging funding fees. If we exclude them someone could miss out on paying/earning funding fees if they are rolling over or resizing when this function is called.